### PR TITLE
Auto-select country entry.

### LINKED ISSUE #1265: Use an autocompleting entry for Country
Using a dropdown for country is kind of painful. If would be nicer if it were an entry with autocompletion. We could use this: https://github.com/JamieAppleseed/selectToAutocomplete

### PR DESCRIPTION
**Pull request by lewisgoddard**
Improves upon #1265 Use an autocompleting entry for Country

### Changes Summary

- Detect the country.
- Select it automatically.
- Trigger an update.

This pull request is ready for review.

### CODE REVIEW

**Comment by lewisgoddard** [comment]
How do we want this to deal with no country detected (hardly ever happens). Currently defaults to the first in list.

File: backend/classify.functions.php [hunk 1]
```diff
@@ -130,3 +130,30 @@ function ipHash($hostname, $debug = false) {
...
+        }
+        $country   = $record->country->isoCode;
+
+    } catch (\Exception $e) {
+        echo '<!-- '.$e->getMessage().' -->'."\n";
+        $country   = false;
+    }
+
+    if ( $debug ) {
+        echo 'Country: "'.$country.'"'."\n";
```
Comment (btkostner) [inline backend/classify.functions.php]: Why are we echoing data? Isn't it possible to leak sensitive data in the error?

Comment (lewisgoddard) [inline backend/classify.functions.php]: @btkostner That file has always done that, and only with the debug mode enabled, it's what you see at https://elementary.io/backend/classify.get.php

There's no sensitive data there, only the individuals detected country/region.

**Comment by danrabbit** [comment]
If we're using an entry instead of a combobox, it should be blank so users can see the hint text.

If this branch doesn't intend to use the entry and is just about automatically grabbing the country, I would default to whatever our country with the most visits is (I'm gonna guess US or maybe Brazil)

**Comment by lewisgoddard** [comment]
@danrabbit This isn't an entry, just an auto-selector, so keep US?

**Comment by lewisgoddard** [comment]
@danrabbit I've modified the title for you.

**Pull request merged**